src/layout/nav/Header.tsx

```diff
@@ -24,17 +25,37 @@ export type NavHeaderProps = {
     // for handling the closeAll action is within the parent object, and will
     // have an effect on navList property.
     onCloseAll?: () => void;
-
     primaryShown: boolean;
-
     secondaryShown: boolean;
+    style?: any;
+    color?: HeaderColor;
+    className?: string;
 };
+export enum HeaderColor {
+    HEADER1 = "header-1",
+    HEADER2 = "header-2",
+    HEADER3 = "header-3",
+    HEADER4 = "header-4",
+    HEADER5 = "header-5",
+    HEADER6 = "header-6",
+    HEADER7 = "header-7",
+}
 
 export class Header extends React.PureComponent<NavHeaderProps> {
```

Comment:
Probably it would be good to add defaultProps for style, color, className cause they can be underfined